src/web/app/page.component.ts

```diff
@@ -79,6 +81,11 @@ export class PageComponent {
           this.pageTitle = resp.pageTitle;
           this.title.setTitle(resp.htmlTitle || DEFAULT_TITLE);
         });
+        this.route.queryParams.subscribe((queryParams: any) => {
```

Comment:
The subscribing binding should not be inside the subscriber of `this.router.events` right?